recipes/sasl/yum-requirements.txt

```diff
@@ -0,0 +1 @@
+libsasl2-dev
```

Comment:
I am not sure that is the correct name for it on CentOS. Ideally we should package that on conda-forge to avoid relying on `yum-requirements.txt`. Note also that, if that lib is needed at run-time, the package will be broken even with `yum-requirements.txt`.